### ISSUE TITLE
limitation: screenshots and location tracking is delayed

### ISSUE THREAD

**Issue by chrisgraf** [boku-ilen/landscapelab#34]
tracking starts quite some time after the play button was pressed and is not stopped with pause/stop button?

**Comment by chrisgraf** [comment]
may have to do with low priority in the thread handler ?

**Comment by kb173** [comment]
In 93d432e, the script was changed so it gets the coordinates / screenshot without delay - only after that, a task is put in a ThreadPool to actually send/save that data. That task also has a medium priority now (as opposed to 0 before).

@chrisgraf, can you verify it is now working as intended?

**Comment by chrisgraf** [comment]
currently it results in error messages .. the number of arguments seems to be wrong:
![image](https://user-images.githubusercontent.com/1245084/64126193-24af1100-cdad-11e9-9bb2-70f7f995c897.png)

**Comment by kb173** [comment]
Can't reproduce the error for some reason, but is it working with 0dc1ad2?

**Comment by chrisgraf** [comment]
Screenshots should be found in the godot users:// directory (Appdata/Roaming/Godot/user_projects/LandscapeLab

**Comment by kb173** [comment]
Screenshots and server requests work and have the correct timestamp
